### ISSUE TITLE
Step 5 and 6 for point cloud modification of kettle.ply in CloudCompare

### ISSUE THREAD

**Issue by monajalal** [liuyuan-pal/NeRO#10]
Hi there very impressive work. I was very excited to see the point cloud after run_colmap.py and also segmenting it. However, I find the instruction on 5 and 6 specially 5 hard to follow. Would you please be able to create a video cast to show how you do step 5 and 6?

I was not able to do step 5 but for step 6 when I placed the plan from Tools --> Fit --> Plane, it wasn't like yours. Can you please share a more detailed tip how you did yours?
![Screenshot from 2023-06-16 09-47-13](https://github.com/liuyuan-pal/NeRO/assets/1892917/fff2c72e-5187-4a4b-a925-c8207dc8f780)
![Screenshot from 2023-06-16 09-47-55](https://github.com/liuyuan-pal/NeRO/assets/1892917/3ab14063-7925-47ff-bcc8-5920756100bc)

**Comment by liuyuan-pal** [comment]
Hi, on step 5, I first crop a subset of the point cloud on the board
![image](https://github.com/liuyuan-pal/NeRO/assets/23974453/d33c56db-b004-4d54-99cd-1429025862c9)
![image](https://github.com/liuyuan-pal/NeRO/assets/23974453/0c52c576-736b-40f2-a97d-6238495d7cfc)
Then, the plane fitting is only conducted on the cropped subset.

**Comment by monajalal** [comment]
@liuyuan-pal thanks a lot for your response. Could you please have a look and let me know why my segmentation is not working? It shows it as segmented but when I click on the green checkmark, it doesn't crop it.
Any pointer is really appreciated.
[kettle_segmentation.webm](https://github.com/liuyuan-pal/NeRO/assets/1892917/205b2b30-cf0c-4133-a3b9-d277f0c1fb42)

**Comment by liuyuan-pal** [comment]
Your segmentation here is OK. You just need to choose the segmented one and then fit a plane on it.

BTW, your segmentation includes some extra points in the object point cloud (very far away from it), which enlarges the bounding box. 
![image](https://github.com/liuyuan-pal/NeRO/assets/23974453/5fb7170e-8f71-42f3-bdef-66934bc24e4e)
![image](https://github.com/liuyuan-pal/NeRO/assets/23974453/64349d31-df44-43e6-bfe4-117026761981)

It is important to not include extra outlier points because we use this point cloud to define the bounding sphere for the object.

I'll make a tutorial screenshot video next week.

**Comment by monajalal** [comment]
@liuyuan-pal Hi Yuan I was wondering if you could share your screencast showing these stages?

**Comment by liuyuan-pal** [comment]
Hi, two videos are here
https://connecthkuhk-my.sharepoint.com/:v:/g/personal/yuanly_connect_hku_hk/EYRhCQmWUqNIgLIFGvGrRkIBGaffGCW7QNIx1HqwWz1rig?e=zffNzi

https://connecthkuhk-my.sharepoint.com/:v:/g/personal/yuanly_connect_hku_hk/EYRhCQmWUqNIgLIFGvGrRkIBeqH5reMI_DM91hiWSPcz6A?e=pmo1Mh

**Comment by liuyuan-pal** [comment]
LMB means Left-Mouse-Button, RMB means Right-Mouse-Button and MMB means Middle-Mouse-Button.
